lib/locales/pl.yml

```diff
@@ -19,9 +19,10 @@ pl:
   eoc_solutions_title: Rozwiązania
   eob_index_title: Skorowidz rzeczowy
   eob_index_symbols_group: Symbole
-  references: Bibliografia
+  references: Bibliografia # is this used somewhere outside eoc?
   eoc:
     glossary: Kluczowe pojęcia
     key-equations: Najważniejsze wzory
     summary: Podsumowanie
+    references: Bibliografia
```

Comment:
As I write in my previous comment in pl-psychology References is not EOC section, so we can remove this :slightly_smiling_face:

Reply:
I believe that `eoc.references` is used by `BakeChapterReferences`, which I refactored in this PR. The `EocCompositePageContainer` direction requires that the title use the translation key `eoc.{container_key}` where `container_key` comes from the `os-{container_key}-container` wrapper class. So it seems that both `references` and `eoc.references` are necessary now.

So I'll take out the comment & leave everything else in, thank you for clarifying!